daemon/api_apps.go

```diff
@@ -54,9 +56,27 @@ var (
 	}
 )
 
+var newStatusDecorator = func(ctx context.Context, isGlobal bool, uid string) clientutil.StatusDecorator {
+	if uid == "0" || isGlobal {
```

Comment:
silly question, what if the user is root and they want to see the status of their services?

Reply:
I wonder the same, shouldn't root be allowed to use something like "--user" to get to its user services?

Reply:
Hmm then I'd prefer identical behaviour that always returns the status of the services for the current user (root included), with the option to use `--global`

Reply:
This is now handled